tests/lib/snaps/test-snapd-eds/snap/snapcraft.yaml

```diff
@@ -0,0 +1,47 @@
+name: test-snapd-eds
+version: '0.1'
+summary: utilities to test evolution-data-server interfaces of snapd
+description: |
+  This package contains utilities designed to test a confined
+  application's ability to access addressbooks and calendars managed
+  by evolution-data-server.
+
+grade: stable
+confinement: strict
+
+apps:
+  contacts:
+    command: bin/contacts
+    plugs:
+      - desktop-contacts-service
+  calendar:
+    command: bin/calendar
+    plugs:
+      - desktop-calendar-service
+
+environment:
```

Comment:
This whole test brings a lot of interesting insight into how the g* stack works. Thank you!